### DISCUSSION TITLE
Slim application Error

### DISCUSSION THREAD

**Discussion by abhinav-TB** [nextcloud/all-in-one#784] [Questions] [answered]
### Steps to reproduce
1.run
```sh
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest-arm64
```
2. Add DNS record to the server
3. visit domain.tld:8443

### Expected behavior 
show nextcloud AIO interface
### Actual behavior 
![image](https://user-images.githubusercontent.com/54744701/172062619-0b47da1a-2941-42f8-88a0-c34ebd1318b9.png)

### Host OS 
Ubuntu 20.04(ARM)

#### Nextcloud AIO version 
latest
#### Current channel 
latest

**Comment by szaimen** [comment]
Hi, can you post the output of `sudo docker inspect nextcloud-aio-mastercontainer` ?

**Comment by abhinav-TB** [comment]
> Hi, can you post the output of `sudo docker inspect nextcloud-aio-mastercontainer` ?

[
    {
        "Id": "1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8",
        "Created": "2022-06-05T16:55:00.68641588Z",
        "Path": "start.sh",
        "Args": [
            "/usr/bin/supervisord",
            "-c",
            "/supervisord.conf"
        ],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 12907,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2022-06-05T17:14:54.486187776Z",
            "FinishedAt": "2022-06-05T17:14:52.46948247Z"
        },
        "Image": "sha256:d180881053cbf1b35216538a7cea6db90c3668a72a9ac653eb7f0dc5f7f7d249",
        "ResolvConfPath": "/var/lib/docker/containers/1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8/hostname",
        "HostsPath": "/var/lib/docker/containers/1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8/hosts",
        "LogPath": "/var/lib/docker/containers/1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8/1bcfce9674f50a2590177581284a0de57c04762db1c487d65562f596defbd3b8-json.log",        "Name": "/nextcloud-aio-mastercontainer",
        "RestartCount": 1,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "nextcloud_aio_mastercontainer:/mnt/docker-aio-config",
                "/var/run/docker.sock:/var/run/docker.sock:ro"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "default",
            "PortBindings": {
                "80/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "80"
                    }
                ],
                "8080/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "8080"
                    }
                ],
                "8443/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "8443"
                    }
                ]
            },
            "RestartPolicy": {
                "Name": "always",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "private",
            "Dns": [],
            "DnsOptions": [],
            "DnsSearch": [],
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": [],
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": [],
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/c6514a3246da710e3047917a25cfee491b35ee5b3d3e201355dc126f86720387-init/diff:/var/lib/docker/overlay2/3a7e9336351039ceac7e962c8f3a705b170fd408960d9d71dbdbbe1dcaba345e/diff:/var/lib/docker/overlay2/a06284132a5fa39d63184e9c52fc4bc80bb5b976701e1df682e179556aa530fa/diff:/var/lib/docker/overlay2/c77caacc8f7b03ff0b419e91d1b143c97368d80c5a7dad9ccf1be957529ff5a0/diff:/var/lib/docker/overlay2/a9404aca486e57f834015397ebba1a8dfe58c6fee93c8cb5c6c7b2b455f4452d/diff:/var/lib/docker/overlay2/e496a9e5d2c460752db2cfeadbb67641a5096c4d59568b43dc84dfd7db90a646/diff:/var/lib/docker/overlay2/a5955bd5bb654d2fc8396f04fa709601e0ce5457c0e6e56ad296d9df4d745e6f/diff:/var/lib/docker/overlay2/da28ff3f21eb2b7749777cc096deec8238330791f4d503548071731816207409/diff:/var/lib/docker/overlay2/f2066d57816dd71f2bd3039ce5a0eaaf9f005e5707a0ba30d88948e1e1637628/diff:/var/lib/docker/overlay2/94af4cda042628521fa20b20abd85f91648237dfc0035ac2dc988c90c4455b6a/diff:/var/lib/docker/overlay2/8b452b9f7c1eeb4189ca25e07170780f7cc266fb859092091f71a1b836938c13/diff:/var/lib/docker/overlay2/c5608d3e10b65314e0199bfa3d9d43f44503ec987a416947a265501a43fc2977/diff:/var/lib/docker/overlay2/e83d1f1e24e0f68c22d72b1d2470fbba373ee479d30d5a2b58afd19a4c90e005/diff:/var/lib/docker/overlay2/fff4994180208a2bd83e85076f443bcc219a1580738e8decc158cac65065d04f/diff:/var/lib/docker/overlay2/0bd7ec4687d08044de3e7f63e19f42e987828ac0ac443dbe41c9d2c9e079c831/diff:/var/lib/docker/overlay2/e06d6bc87992b42faf0b55762576c8e8602df5da39f93385efae4c0f54f77d81/diff:/var/lib/docker/overlay2/3fc21fb782a22b84c1b64693c09448257c1ac6ba843a38104d9fcde7f1c53822/diff:/var/lib/docker/overlay2/0d480df3821c66ac835316efc9ddbf27a9f4630c83c6cb2d0826a4255a49c8d7/diff:/var/lib/docker/overlay2/b7d0ff357c5114caba253875cbb7528276979220cac95195b57bddf2e33e3839/diff:/var/lib/docker/overlay2/0f631a46dab1c0c7f178efc15718e1a650c060e50c51c7257bee68b91850973a/diff:/var/lib/docker/overlay2/adc392ef1c568b38fd62927946324d1e57b95f29ff68815cdfec772a062f7f79/diff:/var/lib/docker/overlay2/16bc73a156ea3a9c8cd40e2ed0864b1da80ef4ed436411ad04344f22d6b39ff0/diff:/var/lib/docker/overlay2/d6b43dc0a3e623a84b1a4c22c38961e63a7a6d1b68e8e7329ec848a5145430c4/diff:/var/lib/docker/overlay2/2c05e5bef5ca7482fe69c4780e0c77234da81841a034952261271a5324979798/diff:/var/lib/docker/overlay2/96b9d870af8041c59185f772cc235c59693e0ba18f1f8a8a90ed0b3a76f12db2/diff:/var/lib/docker/overlay2/2562913d96754a77ca49135ee1bb4aff8841c03e28d1f91135927c795b4acebd/diff:/var/lib/docker/overlay2/d08beb717477c3cdd4dbddc076c627ed6785c2857e45e25409f892b6c310bf43/diff:/var/lib/docker/overlay2/2a28b3b0bc6cc783b795768ac8e0453827bc7f0f869bd8581030db6c6a2331e2/diff:/var/lib/docker/overlay2/d05214e38fa38e910ae5b2bc290152dba8df1c206f608d128220e5b49a466767/diff:/var/lib/docker/overlay2/5c7c1bc0274a64df9dcacd3eb3dec4cce8bee889281e1be316a5577eeda40d60/diff:/var/lib/docker/overlay2/8b97124201a312e40397463fa941c57aa717a6963c5d87c150514b6d3f134e9d/diff:/var/lib/docker/overlay2/391d864d3207162d519669c48c3499998772216c133f3fc5c7814d7994e185b7/diff:/var/lib/docker/overlay2/f53ff7a75cc36087ba20b6c08563ba4fb0b21b245699fe69c53a461dc3c8102e/diff:/var/lib/docker/overlay2/59c056ea5a3762ab14b86998ea984c40156e0ae0e9dca2e784d1bc8115579cc4/diff:/var/lib/docker/overlay2/a5ec7b1c37af570c2566afb4c53e64b1836c28b29580c1b65805fad9b4ce4620/diff:/var/lib/docker/overlay2/291bacb09b9c7bd4c1ef5e931777ccdea3f067d61b3f5a09f383c259f214b7d3/diff:/var/lib/docker/overlay2/e50495ce66f4c07cdfd5b41062c1b100b7da4a625189222a8847e5cdb814cc24/diff",
                "MergedDir": "/var/lib/docker/overlay2/c6514a3246da710e3047917a25cfee491b35ee5b3d3e201355dc126f86720387/merged",
                "UpperDir": "/var/lib/docker/overlay2/c6514a3246da710e3047917a25cfee491b35ee5b3d3e201355dc126f86720387/diff",
                "WorkDir": "/var/lib/docker/overlay2/c6514a3246da710e3047917a25cfee491b35ee5b3d3e201355dc126f86720387/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "volume",
                "Name": "nextcloud_aio_mastercontainer",
                "Source": "/var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data",
                "Destination": "/mnt/docker-aio-config",
                "Driver": "local",
                "Mode": "z",
                "RW": true,
                "Propagation": ""
            },
            {
                "Type": "bind",
                "Source": "/var/run/docker.sock",
                "Destination": "/var/run/docker.sock",
                "Mode": "ro",
                "RW": false,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "1bcfce9674f5",
            "Domainname": "",
            "User": "root",
            "AttachStdin": true,
            "AttachStdout": true,
            "AttachStderr": true,
            "ExposedPorts": {
                "80/tcp": {},
                "8080/tcp": {},
                "8443/tcp": {}
            },
            "Tty": true,
            "OpenStdin": true,
            "StdinOnce": true,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "PHPIZE_DEPS=autoconf \t\tdpkg-dev \t\tfile \t\tg++ \t\tgcc \t\tlibc-dev \t\tmake \t\tpkg-config \t\tre2c",
                "PHP_INI_DIR=/usr/local/etc/php",
                "APACHE_CONFDIR=/etc/apache2",
                "APACHE_ENVVARS=/etc/apache2/envvars",
                "PHP_CFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64",
                "PHP_CPPFLAGS=-fstack-protector-strong -fpic -fpie -O2 -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64",
                "PHP_LDFLAGS=-Wl,-O1 -pie",
                "GPG_KEYS=1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F",
                "PHP_VERSION=8.0.19",
                "PHP_URL=https://www.php.net/distributions/php-8.0.19.tar.xz",
                "PHP_ASC_URL=https://www.php.net/distributions/php-8.0.19.tar.xz.asc",
                "PHP_SHA256=ba62219c4b0486cbb2a04f0796749a46b0ee1f5a142ed454212b4e2460cb0fab"
            ],
            "Cmd": [
                "/usr/bin/supervisord",
                "-c",
                "/supervisord.conf"
            ],
            "Image": "nextcloud/all-in-one",
            "Volumes": {
                "/mnt/docker-aio-config/": {}
            },
            "WorkingDir": "/var/www/docker-aio",
            "Entrypoint": [
                "start.sh"
            ],
            "OnBuild": null,
            "Labels": {},
            "StopSignal": "SIGWINCH"
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "82e359a833103afe9e47d7fb77b5e5bc9cea78a4ef1438210f201f4b76e00800",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {
                "80/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "80"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "80"
                    }
                ],
                "8080/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "8080"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "8080"
                    }
                ],
                "8443/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "8443"
                    },
                    {
                        "HostIp": "::",
                        "HostPort": "8443"
                    }
                ]
            },
            "SandboxKey": "/var/run/docker/netns/82e359a83310",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "b4579200e38a4372b1a601a03f44dc3cc2868428b28f1706199776e16664eb70",
            "Gateway": "172.17.0.1",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "172.17.0.2",
            "IPPrefixLen": 16,
            "IPv6Gateway": "",
            "MacAddress": "02:42:ac:11:00:02",
            "Networks": {
                "bridge": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": null,
                    "NetworkID": "69e8a5d61cce2e95b325b2f7c7b14a3eca2c8f920415e54ca7180d35f05b0aca",
                    "EndpointID": "b4579200e38a4372b1a601a03f44dc3cc2868428b28f1706199776e16664eb70",
                    "Gateway": "172.17.0.1",
                    "IPAddress": "172.17.0.2",
                    "IPPrefixLen": 16,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "02:42:ac:11:00:02",
                    "DriverOpts": null
                },
                "nextcloud-aio": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": [
                        "1bcfce9674f5"
                    ],
                    "NetworkID": "4945f93ff6dd296a3590ac5c11614f61bffbccc9dcc07784ea078a663b24f9c4",
                    "EndpointID": "1454ffe3dbf2189be1c2871cabe640bfce1830ecc6925dd5576e8fade32c88b1",
                    "Gateway": "172.18.0.1",
                    "IPAddress": "172.18.0.2",
                    "IPPrefixLen": 16,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "02:42:ac:12:00:02",
                    "DriverOpts": null
                }
            }
        }
    }
]

**Comment by szaimen** [comment]
Thanks! Can you please try to run `sudo docker stop nextcloud-aio-mastercontainer`, `sudo docker rm nextcloud-aio-mastercontainer` and then recreate it with 

```
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest-arm64
```

**Reply by abhinav-TB** [reply]
@szaimen thanks for your help, but I switched to a x64 VM now, still getting the same error.

**Comment by abhinav-TB** [comment]
this is the docker log for the slim error page
```
localhost - - [06/Jun/2022:05:00:23 +0000] "GET /containers HTTP/1.1" 200 1610 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36"
[Mon Jun 06 05:00:25.663692 2022] [php:notice] [pid 134] [client 127.0.0.1:59514] 404 Not Found\nType: Slim\\Exception\\HttpNotFoundException\nCode: 404\nMessage: Not found.\nFile: /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php\nLine: 91\nTrace: #0 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(72): Slim\\Middleware\\RoutingMiddleware->performRouting(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#1 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(456): Slim\\Routing\\RouteRunner->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#2 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(209): Slim\\Csrf\\Guard->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Slim\\Routing\\RouteRunner))\n#3 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(147): Slim\\Views\\TwigMiddleware->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#5 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(313): AIO\\Middleware\\AuthMiddleware->__invoke(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#7 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(107): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(147): Slim\\Middleware\\ErrorMiddleware->process(Object(GuzzleHttp\\Psr7\\ServerRequest), Object(Psr\\Http\\Server\\RequestHandlerInterface@anonymous))\n#9 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(81): Psr\\Http\\Server\\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#10 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(215): Slim\\MiddlewareDispatcher->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#11 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\\App->handle(Object(GuzzleHttp\\Psr7\\ServerRequest))\n#12 /var/www/docker-aio/php/public/index.php(156): Slim\\App->run()\n#13 {main}, referer: https://cloud.abhinavtb.com:8443/containers
172.17.0.2:8000 localhost - - [06/Jun/2022:05:00:25 +0000] "GET /favicon.ico HTTP/1.1" 404 668 "https://cloud.abhinavtb.com:8443/containers" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36"
localhost - - [06/Jun/2022:05:00:25 +0000] "GET /favicon.ico HTTP/1.1" 404 668 "https://cloud.abhinavtb.com:8443/containers" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36"
```

**Reply by szaimen** [reply]
This seems to be a different issue now?

**Reply by abhinav-TB** [reply]
> 

nope, same issue, i just pasted the docker logs aswell
![image](https://user-images.githubusercontent.com/54744701/172124148-a90b48be-19a1-4d21-8451-427f67ff5271.png)

**Reply by szaimen** [reply]
I see. Can you post the output of `sudo docker version` here?

**Reply by abhinav-TB** [reply]
> sudo docker version

ubuntu@next-cloud-887397:~$ sudo docker version
Client:
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.17.3
 Git commit:        20.10.12-0ubuntu4
 Built:             Mon Mar  7 17:10:06 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.3
  Git commit:       20.10.12-0ubuntu4
  Built:            Mon Mar  7 15:57:50 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.5.9-0ubuntu3
  GitCommit:        
 runc:
  Version:          1.1.0-0ubuntu1
  GitCommit:        
 docker-init:
  Version:          0.19.0
  GitCommit:

**Reply by szaimen** [reply]
Did you use the following command to start the container? (or did you leave out the `latest` tag?)
```
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
```

**Reply by szaimen** [reply]
I could just reproduce this by leaving out the `latest` tag. However when running the correct command it works.

**Reply by abhinav-TB** [reply]
> I could just reproduce this by leaving out the `latest` tag. However when running the correct command it works.

I had the latest image but I tried removing all images and tried running the command you provided and it worked !!!
thanks a lot for your time :)

**Comment by abhinav-TB** [comment]
A quick question is it possible to free ports 80 and 443 to run different services, after nextcloud AIO installation

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

**Reply by szaimen** [reply]
can you post the logs of the apache container here? (just click on `Running` next to Apache)

**Reply by abhinav-TB** [reply]
> can you post the logs of the apache container here? (just click on `Running` next to Apache)

nvm, fixed it was a problem with Cloudflare, it won't allow force SSL, which was an option mentioned in the AIO reverse proxy docs

**Reply by jonnyparris** [reply]
hey @abhinav-TB - how did you resolve this: did you just abandon/replace Cloudflare? Or some other config tweak?

**Reply by jonnyparris** [reply]
nvm, fixed itself after a reload. I think it might have just needed some time to propagate changes

**Comment by szaimen** [comment]
Fixed with

> Did you use the following command to start the container? (or did you leave out the `latest` tag?)
> 
> ```
> sudo docker run -it \
> --name nextcloud-aio-mastercontainer \
> --restart always \
> -p 80:80 \
> -p 8080:8080 \
> -p 8443:8443 \
> --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
> --volume /var/run/docker.sock:/var/run/docker.sock:ro \
> nextcloud/all-in-one:latest
> ```

**Comment by Shahin-rmz** [comment]
Hello every one.
I am facing same error on my ARM64 Raspberry pi 4.
I use this command:
```
sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 8090G:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest
```
I am checking `http://internalip:8080` and here is output of docker inspect

```
[
    {
        "Id": "66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb",
        "Created": "2023-06-22T12:20:38.199283076Z",
        "Path": "/start.sh",
        "Args": [
            "/usr/bin/supervisord",
            "-c",
            "/supervisord.conf"
        ],
        "State": {
            "Status": "running",
            "Running": true,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 16034,
            "ExitCode": 0,
            "Error": "",
            "StartedAt": "2023-06-22T12:28:22.218985244Z",
            "FinishedAt": "2023-06-22T12:28:19.686242018Z",
            "Health": {
                "Status": "healthy",
                "FailingStreak": 0,
                "Log": [
                    {
                        "Start": "2023-06-22T15:26:40.871659725+03:00",
                        "End": "2023-06-22T15:26:41.000943152+03:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-06-22T15:27:11.014492456+03:00",
                        "End": "2023-06-22T15:27:11.153984976+03:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-06-22T15:27:41.164813486+03:00",
                        "End": "2023-06-22T15:27:41.298285083+03:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-06-22T15:28:11.309332562+03:00",
                        "End": "2023-06-22T15:28:11.478103772+03:00",
                        "ExitCode": 0,
                        "Output": ""
                    },
                    {
                        "Start": "2023-06-22T15:28:52.220094695+03:00",
                        "End": "2023-06-22T15:28:52.390627922+03:00",
                        "ExitCode": 0,
                        "Output": ""
                    }
                ]
            }
        },
        "Image": "sha256:8acbe6852aa6976858e0175a3775e9f2c04258cfc419b8d14d77fba81325e350",
        "ResolvConfPath": "/var/lib/docker/containers/66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb/hostname",
        "HostsPath": "/var/lib/docker/containers/66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb/hosts",
        "LogPath": "/var/lib/docker/containers/66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb/66ea4475de56c1ab33ef9731fbb511dd77498ae360c993cddca8bf18fcca44eb-json.log",
        "Name": "/nextcloud-aio-mastercontainer",
        "RestartCount": 1,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "nextcloud_aio_mastercontainer:/mnt/docker-aio-config",
                "/var/run/docker.sock:/var/run/docker.sock:ro"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "default",
            "PortBindings": {
                "80/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "8090"
                    }
                ],
                "8080/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "8080"
                    }
                ],
                "8443/tcp": [
                    {
                        "HostIp": "",
                        "HostPort": "8443"
                    }
                ]
            },
            "RestartPolicy": {
                "Name": "always",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": null,
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "private",
            "Dns": [],
            "DnsOptions": [],
            "DnsSearch": [],
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": [],
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": [],
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/fd13d5e9e6223a8da07562e6e3dc509ad6bda32e7c389044033308a175a9c699-init/diff:/var/lib/docker/overlay2/4f8ce5ef2f7a1708179799be64f168ac43cc21a4f5c9077ca23596e6326702b5/diff:/var/lib/docker/overlay2/85e7d3a48de7ecf0d3adeb376fccd1d79f4fb14f95fa9bfc3b2bda4902988b6e/diff:/var/lib/docker/overlay2/ee87ff9c2f1b40de07b15a23ac6bc77acbd5f8a0b1507c0bd9e5faed633c39fb/diff:/var/lib/docker/overlay2/71f39a51484505ce1f02ca73d83d3e14b520ce4af8df32a178304ad5543faf6e/diff:/var/lib/docker/overlay2/3edaec4dd02e6110241f5b3b7dfae824bfd6399775ca84bae47d1847bfb941f3/diff:/var/lib/docker/overlay2/38ada4b9a849d8b04f7dfc79610ad75e019573106e942c79ca649675251c867a/diff:/var/lib/docker/overlay2/5e5db806514d3f9d302eb41a51f7157f529aae1017a9ce9da3fa48be69eaf062/diff:/var/lib/docker/overlay2/50159f80a108941cbeef09716ba5fff57f9946ecbb0f8a8c944a5997a72b21f9/diff:/var/lib/docker/overlay2/47ce4e6624f8681e1b24fd7e96b01bc69aab318265aa8ca8a90d88b537f37be3/diff:/var/lib/docker/overlay2/a16ee3e1cf9bbab5e80e44a7eb4a58388427e5e1ab1ec91c1f260e65773efab2/diff:/var/lib/docker/overlay2/8ed0b61a08a3177884b2c17cde360a41a53a97fa3d620f375d9520d7e2333200/diff:/var/lib/docker/overlay2/c1a112970c7ea5cd8531742e624865f6e213da3033add40fa4c45dde3d5ddf5c/diff:/var/lib/docker/overlay2/f916e0a8f80e6f142e18f8f14fafa659a22342e21eb79d4
                
                
                
                
```
(I am not sure if this is complete output of `docker inspect`.
Thanks for help.

**Reply by szaimen** [reply, marked as answer]
Hi, please create a new thread for this here: https://github.com/nextcloud/all-in-one/discussions/new?category=questions
Thanks!
